src/bases/DataProviderBase.ts

```diff
@@ -55,6 +55,14 @@ abstract class DataProviderBase<T, O extends Options> {
 			if (this._data) {
 				observer.next(this._data);
 			}
+			return {
+				unsubscribe: () => {
```

Comment:
Should this really be in the format of a `Handle`?  I agree that `destroy()` doesn't accurately represent it, but it ensure a level of consistency across the framework and long term maintainability.